shop/mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
Please remove all these files regarding mvn (as written in the Training Chapter 2). Or just exclude them from git using .gitignore

Reply:
Fixed